github-link.php

```diff
@@ -105,5 +108,18 @@ function GHL_plugin_link( $actions, $plugin_file, $plugin_data, $context ) {
         }
     }
 
+
+    // if from wordpress.org, show that now.
+    if ( ! empty( $plugin_data['url']) && strstr($plugin_data['url'],'wordpress.org') !== FALSE) {
```

Comment:
WP coding style please:
```php
false !== strstr( $plugin_data['url'], '//wordpress.org/plugins/' ) )
```
and split the two conditions into two lines as above.